recipe/meta.yaml

```diff
@@ -29,6 +29,10 @@ requirements:
     - python-confluent-kafka  # [not win]
     - redis-py
     - slack-sdk
+    - globus-sdk
+    - appdirs
+    - pyzbar
+    - py4xs
```

Comment:
This is already included in the [`nsls2-analysis`]( https://github.com/nsls-ii-forge/nsls2-analysis-feedstock/blob/d4684602c67b3829e27219b256d7bc857bf30852/recipe/meta.yaml#L75).